src/registrar/models/dns/dns_account.py

```diff
@@ -1,9 +1,27 @@
+import logging
+
 from django.db import models
 from ..utility.time_stamped_model import TimeStampedModel
+from registrar.models.dns.dns_account_vendor_dns_account import DnsAccount_VendorDnsAccount as AccountsJoin
+
+logger = logging.getLogger(__name__)
 
 
 class DnsAccount(TimeStampedModel):
     name = models.CharField(unique=True, max_length=255)
     vendor_dns_account = models.ManyToManyField(
         "registrar.VendorDnsAccount", through="DnsAccount_VendorDnsAccount", related_name="accounts"
     )  # type: ignore
+
+    def get_active_x_account_id(self):
```

Comment:
@abe-alam-ecs - Erin and I talked about this and realized it probably was more appropriate to be a getter than a property. Looking at the usages of property, it's more for something that might change based on other attributes on the model, which is not true in this case. It was also more complex with the error handling than would normally be found in a property. So I made this a method instead of a property.